komorebi-wallpaper-creator.in

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import argparse
+import gi
+import logging
+import sys
+import os
+
+gi.require_versions({
+	'Clutter': '1.0',
+	'Gtk': '3.0'
+})
+
+from gi.repository import Gtk, Gio
+
+import komorebi.consts
+
+from komorebi.wallpaper_creator.window import WallpaperWindow
+
+def main_parser():
+	parser = argparse.ArgumentParser()
+	parser.add_argument('-v', '--version',
+						action='store_true',
+						help='show current version')
+	parser.add_argument('-l', '--log',
+						type=str,
+						choices=['NORMAL', 'INFO', 'DEBUG'],
+						default='NORMAL',
+						help="set logging level for komorebi")
+
+	return parser
+
+def main():
+	print(f'Welcome to {komorebi.consts.package_name} Wallpaper Creator')
+
+	parser = main_parser()
+	args = parser.parse_args()
+
+	if args.version:
+		print(f'Version: {komorebi.consts.package_version}\nMaintained by: Komorebi Team\n')
+		return
```

Comment:
Use the argparse `version` action.

https://docs.python.org/3/library/argparse.html#action

Example: https://github.com/pypa/build/blob/28a67420b18d2287c309500d8aa0b65b74b3bd25/src/build/__main__.py#L122

Reply:
Fixed